starters/lit-mobx-vaadin/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "singleQuote": true,
+    "arrowParens": "avoid",
+    "trailingComma": "always"
+}
```

Comment:
We should make use of the global prettier config located here: https://github.com/thisdot/starter.dev/blob/main/.prettierrc

Reply:
It seems that every package is treated independently.

See:
- https://github.com/thisdot/starter.dev/blob/main/starters/angular-apollo-tailwind/.prettierrc
- https://github.com/thisdot/starter.dev/blob/main/starters/qwik-graphql-tailwind/.prettierrc

I think it makes sense because when using the starters the user won't have access to the root setup.